#! /usr/bin/env python
# Create a cfg file for getting control parameters for dmoc optimal controller

PACKAGE = 'gcop_ros_bullet'

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

#       Name          				Type      Reconfiguration level        Description   				Default   			Min   			 Max
gen.add("iterate"     ,     bool_t     ,     0     ,     "Run Nit iterations"            ,     False )
gen.add("send_traj"   ,     bool_t     ,     0     ,     "Send Trajectory"               ,     False )
gen.add("animate"     ,     bool_t     ,     0     ,     "Animate the current solution"  ,     False )
gen.add("Nit"         ,     int_t      ,     0     ,     "Number of iterations of dmoc"  ,     5     , 0   , 30)

exit(gen.generate(PACKAGE , "Config" , "CEInterface"))
